pkg/ddevapp/silverstripe.go

```diff
@@ -0,0 +1,71 @@
+package ddevapp
+
+import (
+	"fmt"
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/nodeps"
+	"github.com/ddev/ddev/pkg/util"
+	"os"
+	"path/filepath"
+)
+
+// TODO Check if location is still correct when composer bin folder is changed.
```

Comment:
Has this TODO been taken care of?

Reply:
No, it is added as per @gilbertsoft his comment higher up, where he asked to add this, so it would be taken care of when the composer bin folder is taken from the composer.json file instead of hardcoded `vendor/bin/sake`.

Reply:
I believe that with the suggested changes of including the composer root in the `filepath.join()` calls, it is resolved, but I'd like confirmation of that from @gilbertsoft , before I remove the todo.

Reply:
You can test manually too... And GetComposerRoot() should help. Note that you will use `filepath.Join()` when operating on the host, and `path.Join()` when doing things inside the container.

Reply:
> so it would be taken care of when the composer bin folder is taken from the composer.json file instead of hardcoded vendor/bin/sake

There is no plan for the composer.json to be parsed or used for the composer bin directory; the way things currently are (`composer_root` determines where we look for vendor, and vendor/bin is automatically added to the PATH) is what I'd expect. No changes in how this is being handled are contemplated.

Reply:
Read the comment, see what you're saying. I'm not as much a fan as Simon about adding TODOs, would rather fix things when the future hoped-for thing goes in.

Reply:
@rfay Just to be clear... when you say "Simon"... do you mean me or @gilbertsoft ... because... my name is Simon.... and from a quick web search... it seems I'm not the only one with that name!

I'm not a big fan of leaving in TODOs. I'm going to remove it, as with the suggested changes from @gilbertsoft , it seems to work okay. If anything breaks, I will revisit when it breaks, instead of now. I'm somewhat assuming I'll notice a pre-release will let me know something is wrong.

Reply:
Sorry! You're Firesphere and @gilbertsoft is Simon. I didn't want to ping him :)

Reply:
All good... a bit of confusion on my side. Simon is too common a name 😆 

I'm happy to stick to being called "Firesphere" 👍 

Just getting the artifact repo up and ready. I'd like to run some tests before I do the final push and make it all "okay". Just to make sure the correct artifacts are being pulled in and all.

I'll update again when I get it all done. Probably tomorrow morning NZ time.